x-pack/plugins/synthetics/server/synthetics_service/get_service_locations.test.ts

```diff
@@ -69,6 +69,7 @@ describe('getServiceLocations', function () {
               lon: -95.86,
             },
             id: 'us_central',
+            isInvalid: false,
```

Comment:
Should we be adding `isInvalid` to service location schema? Seems like a private location only thing.